miden/src/cli/data.rs

```diff
@@ -480,3 +508,90 @@ impl Libraries {
         Ok(Self { libraries })
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
minor nit: could we add our usual section separator above to indicate that this is a start of the "TEST" section?